test/testdata/schema.sql

```diff
@@ -20,6 +20,8 @@ CREATE TABLE FullTypes (
   FTBytes BYTES(32) NOT NULL,
   FTBytesNull BYTES(32),
   FTTimestamp TIMESTAMP NOT NULL,
+  FTCommitTimestampNull TIMESTAMP OPTIONS (allow_commit_timestamp=true),
```

Comment:
ahhh! I changed tests to use a fake spanner server, so this schema does not support this DDL for now. I know this PR but I didn't see this change. I think these columns not affect generated codes, so how about just removing this change for now?
I want the official fake spanner to be able to parse this DDL actually even if it does not understand semantically.

Reply:
`cloud.google.com/go/spanner/spansql` can't parse such columns.
If testdata doesn't include these columns, codes written with `cloud.google.com/go/spanner/spansql` would pass tests.
So, I added these columns for cases that fail in using `cloud.google.com/go/spanner/spansql`.